src/services/card-info/index.js

```diff
@@ -157,6 +188,27 @@ export default class CardInfo extends Card {
         return mask;
     }
 
+    _getAllMasks(maskDigitSymbol, maskDelimiterSymbol, allBlocks) {
```

Comment:
In my opinion, implementation with reduce will be better.

`const cells = [4, 4, 4, 4]

const mask = cells.reduce((prevCell, cellCharsLength, idx) => {
    const cellMask = prevCell + Array(cellCharsLength + 1).join('#');
    const delimiter = ' ';
    return idx < cells.length - 1  ? `${cellMask}${delimiter}` : cellMask;
}, '')`

Reply:
\+